README.md

```diff
@@ -41,14 +41,70 @@ We've put together a very simple [example shell](./shellbase-example) that can g
 
 ## [Dev] Building/testing locally
 
-1. Locally compile/install skipping the PGP signatures: 
+To build project in default Scala version:
+```
+./gradlew build
+```
 
-   ```
-   ./mvnw clean install -Dgpg.skip=true
-   ```
-1. Change your consumer `pom.xml` to depend on the `SNAPSHOT` version generated.
+To build project in any supported Scala version:
+```
+./gradlew build -PscalaVersion=2.12.8
+```
 
-## [Dev] How to release new version
+For testing, change your consumer `pom.xml` or `gradle.properties` to depend on the `SNAPSHOT` version generated.
+
+### [Dev] Managing Scala versions
+
+This project supports multiple versions of Scala. Supported versions are listed in `gradle.properties`.
+- `supportedScalaVersions` - list of supported versions (Gradle prevents building with versions from 
+outside this list)
+- `defaultScalaVersion` - default version of Scala used for building - can be overridden with `-PscalaVersion`
+
+### [Dev] How to release new version
+1. Make sure you have all credentials - access to `Open Source` vault in 1Password.
+    1. Can login as `sumoapi` https://oss.sonatype.org/index.html
+    2. Can import and verify the signing key:
+        ```
+        gpg --import ~/Desktop/api.private.key
+        gpg-agent --daemon
+        touch a
+        gpg --use-agent --sign a
+        gpg -k
+        ```
+    3. Have nexus and signing credentials in `~/.gradle/gradle.properties`
+        ```
+        nexus_username=sumoapi
+        nexus_password=${sumoapi_password_for_sonatype_nexus}
+        signing.gnupg.executable=gpg
+        signing.gnupg.keyName=${id_of_imported_sumoapi_key}
+        signing.gnupg.passphrase=${password_for_imported_sumoapi_key}
+        ```
+2. Remove `-SNAPSHOT` suffix from `version` in `build.gradle`
+3. Make a release branch with Scala version and project version, ex. `shellbase-1.5.2`:
+    ```
+    export RELEASE_VERSION=shellbase-1.5.2
+    git checkout -b ${RELEASE_VERSION}
+    git add build.gradle
+    git commit -m "[release] ${RELEASE_VERSION}"
+    ```
+4. Perform a release in selected Scala versions:
+    ```
+    ./gradlew build publish -PscalaVersion=2.11.8
```

Comment:
I guess you meant `2.11.12`

Reply:
Not really, `2.11.8` was the previous version from Maven `pom.xml` and I made it the default in Gradle. Although I can update it to `2.11.12` - please decide.

Reply:
`2.11.12`